### PR TITLE
remove $locale in favor of Lang::get()

### PR DESCRIPTION
**Pull request by ildyria**
Make use of the Lang facade instead of `$locale`.

### CODE REVIEW

**Review by qwerty287 — CHANGES_REQUESTED** [review]
The frontend submodule uses your laravel 9 port branch.

**Comment by ildyria** [comment]
> The frontend submodule uses your laravel 9 port branch.

oupsie. :')

**Review by qwerty287 — APPROVED** [review]

File: resources/views/layouts/gallery.blade.php [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE HTML>
-<html lang="{{ $locale['language'] }}" style="visibility: hidden;">
+<html lang="{{ str_replace('_', '-', Lang::get_code()) }}" style="visibility: hidden;">
```
Comment (nagmat84) [inline resources/views/layouts/gallery.blade.php:2]: Why this replacement? Previously, `$locale['lanuage']` has been equal to `$lang['language'] = Configs::getValueAsString('lang');` from `IndexController` and I don't see a similar replacement there. So why now?

Comment (ildyria) [reply resources/views/layouts/gallery.blade.php:2]: `Configs::getValueAsString('lang')` returns the same as `Lang::get_code()`.
This is just uniforming with `layouts/app.blade.php`.

**Review by nagmat84 — APPROVED** [review]

**Review by qwerty287 — APPROVED** [review]

**Pull request merged**